### PR TITLE
Secondary users: update notices to indicate a Jetpack account is needed

### LINKED ISSUE #6293: User-only connection banner wording improvement
Currently, if you visit the Jetpack dashboard as a "secondary user" on the site (_not_ the account which connected Jetpack originally), you see a banner prompting you to connect your user account:

screen shot 2017-02-03 at 3 52 15 pm

(note "editor" is my username here). I think we should reword this to something like:

"Get more out of Jetpack, connect to a WordPress.com account"

With the button just saying "Connect".

### PR DESCRIPTION
**Pull request by keoshi**
Fixes #6293

#### Changes proposed in this Pull Request:

* Updates notices for secondary users to indicate a Jetpack account is needed.
* Modifies main banner to incorporate Jetpack branding.
* Adds plans gridicon (Jetpack icon).

#### Before

image

#### After

image

#### Testing instructions:

* Fire up this PR.
* Login to your site using a secondary account (as opposed to the primary one used to connect Jetpack).
* Go to the Dashboard and Settings, and ensure you see the updates notices.

#### Proposed changelog entry for your changes:

* Secondary users: update notices to indicate a Jetpack account is needed.

### CODE REVIEW

**Comment by keoshi** [comment]
@brbrr The full context is in the [initial issue](https://github.com/Automattic/jetpack/issues/6293) this PR fixes, but the important bit is this comment: https://github.com/Automattic/jetpack/issues/6293#issuecomment-375043692

**Comment by brbrr** [comment]
Thanks! Although, I wasn't able to find an explanation of why the decision was made to use `Jetpack account` over `WordPress.com account` wording.

**Comment by keoshi** [comment]
@brbrr We're on the same boat here. :) The discussion happened so long ago that I'm afraid the context was lost in there at some point. But I think that differentiation was a secondary effect of moving from the “connecting” wording into “create an account”.

Even though technically we're creating a WordPress.com account, secondary users might not care to learn about that and just want enable features on their Jetpack site. Introducing more entropy there could be problematic, even though reading about a “Jetpack account” and then jumping into a WP.com account flow is not ideal either.

File: _inc/client/components/jetpack-notices/index.jsx [hunk 1]
```diff
@@ -168,10 +168,11 @@ export class UserUnlinked extends React.Component {
 			return (
 				<div className="jp-unlinked-notice">
 					<JetpackBanner
-						title={ __( 'Connect your account to get the most out of Jetpack' ) }
-						callToAction={ __( 'Connect to WordPress.com' ) }
+						title={ __( 'Jetpack is powering your site, but to access all of its features you’ll need to create a Jetpack account.' ) }
```
Comment (MichaelArestad) [inline _inc/client/components/jetpack-notices/index.jsx]: I think it would be good to have a copy review for this. I think it all looks pretty dang good, though. Nice work!

**Review by MichaelArestad — DISMISSED** [review]
Other than a quick copy review, I think this is good to go.

File: _inc/client/components/jetpack-notices/index.jsx (hunk 1 above)
Comment (keoshi) [reply _inc/client/components/jetpack-notices/index.jsx]: @Automattic/editorial can you help with this, please?

**Comment by michelleweber** [comment]
I'd delete the "Jetpack" in "Jetpack account" and just say "account." It's strongly implied, and the iconography makes it really clear that you're creating an account that is necessary for Jetpack to function. The sentence feels a little stilted with the double Jetpack. Otherwise, looking good!

**Comment by keoshi** [comment]
Thanks, @michelleweber — that makes total sense and helps answering the questions above.

**Review by jeherve — DISMISSED** [review]
It looks good on my end, but I find it a bit confusing that we use 2 different Jetpack icons so close to each other:

screenshot 2018-12-13 at 13 58 42

Is there a reason not to use the usual white on green?

**Comment by keoshi** [comment]
Good catch, @jeherve — do you know if that's using a Gridicon? I didn't see a JP one, so I added the `plans` gridicon, which includes the circle.

**Comment by keoshi** [comment]
Aw no, the JP logo in the JITMs is a PHP function :\

https://github.com/Automattic/jetpack/blob/79c36df260713e549e0584b8911b2e4efbd0f607/class.jetpack-jitm.php#L50-L57

**Comment by jeherve** [comment]
We have an SVG with the logo here:
https://github.com/Automattic/jetpack/blob/dd27e523ed192d5651dec9ddf1e8e4f245ab7f89/_inc/client/components/masthead/index.jsx#L70

We could make it its own little component actually, I think it could be useful!

**Comment by keoshi** [comment]
@jeherve The problem is not the SVG, but that the banner component requires a Gridicon and doesn't even allow to change its size.

https://github.com/Automattic/jetpack/blob/f8078c2cd12ac508334da2fb08e37a92cf283c14/_inc/client/components/banner/index.jsx#L62-L86

I mean, we _could_ change that, but for the sake of keeping this PR short and sweet, I think I'll look for an alternative. Would appreciate any suggestions you may have.

**Review by jeherve — APPROVED** [review]
If you are happy with the current look, then this should be good to go!

screenshot 2018-12-17 at 11 59 34

**Comment by keoshi** [comment]
I'd say it's a reasonably acceptable compromise with what we currently have in the JetpackBanner component.

**Comment by jeffgolenski** [comment]
@jeherve Not directly related to this issue, but is that Performance JITM showing when you're not connected? We should change that. Its irrelevant unless the customer is connected. cc @withinboredom

**Comment by jeherve** [comment]
@jeffgolenski It's a very good point. I committed D22452-code to fix this.
